samples/client/petstore/spring-cloud/src/main/java/io/swagger/model/Pet.java

```diff
@@ -13,10 +13,14 @@
 
 
 
+/**
+ * A pet for sale in the pet store
+ **/
 
 /**
- * Pet
+ * A pet for sale in the pet store
  */
```

Comment:
[diff](https://github.com/swagger-api/swagger-codegen/pull/3865/files/2083c03b28c009e3cfde9056b15f3a9b8a89451f#diff-dc82ea0f80e6fc11df755bfdccfa57e1), [raw file](https://github.com/wing328/swagger-codegen/blob/2083c03b28c009e3cfde9056b15f3a9b8a89451f/samples/client/petstore/spring-cloud/src/main/java/io/swagger/model/Pet.java). It looks like this:

```java
 /**
  * A pet for sale in the pet store
  **/
  
  /**
   * A pet for sale in the pet store
   */
```

(This new review mode doesn't show enough context, nor does it easily link back to the file.)